### ISSUE TITLE
All-zero time series

### ISSUE THREAD

**Issue by mikapfl** [JGuetschow/PRIMAP-hist#71]
In current primap-hist output, we drop all-zero time series (such as F-Gas emissions in categories other than `2`). For some users, it would be easier to work with the data if the all-zero time series were included explicitly. The inconsistency is more visible in the native primap2 format based on xarray, which contains the full data set, and these missing time series are `NaN` instead of the expected `0`.

However, the all-zero time series do not offer much more information, so we would produce some not-really-useful information, especially in the interchange format CSV files. For example, F-Gas emissions in category `1` are zero by definition, so including this information in the dataset by explicitly providing full all-zero timeseries is a pretty inefficient way of communicating this fact. We would have to check if the trade-off is worth it or if there are other methods to do this.

**Comment by JGuetschow** [comment]
I'm thinking about a good way to deal with this, especially as some countries do report 0 for sector - gas combinations which never have emissions data. Currently I remove them, but you're right that this is confusing especially in the native format. On the other hand we can not fill all nan time series with 0 as the sectoral resolution is not the same for all gases, so filling e.g. IPPU subsectors with 0 for f-gases would actually be wrong.
Maybe a possible way is to add covered gases to the sector overview in the data description document.

**Comment by mikapfl** [comment]
Yes, we should definitely add covered gases to the sector overview in the data description, and in the native format we can also add another Data Variable calling it "covered" or "defined" or so and it is a boolean. That would help users a lot, I think, because they can easily select only covered combinations.

**Comment by JGuetschow** [comment]
For v2.5 I have added the information on sector - gas coverage to the data description. For the next release we can think of a way to implement the coverage in the data files.
